script.js

```diff
@@ -0,0 +1,63 @@
+const recipe1 = {
+  title: "Pancake",
+  ingredients: ["flour", "baking powder", "sugar", "salt", "milk", "egg"],
```

Comment:
Ingredients should be an array of objects as shown in the example that is in exercise description :)